### DISCUSSION TITLE
Turborepo build output is different than vanilla tsc?

### DISCUSSION THREAD

**Discussion by WesleyYue** [vercel/turborepo#3181] [Help] [answered]
I have a turborepo setup to build a typescript package. When I build with `npx turbo build`, I get the following output:

```
mypackage:build: cache miss, executing a81ccd69d796d509
$ tsc
mypackage:build: src/fileWithError.ts(258,20): error TS2554: Expected 3 arguments, but got 2.
```

The `build` script of `packages/mypackages` is just tsc. When I go to `packages/mypackage` and run `yarn build`, I get:
```
yarn run v1.22.19
$ tsc
src/fileWithError.ts:258:20 - error TS2554: Expected 3 arguments, but got 2.
```

Particularly, the file+line output is different: `src/fileWithError.ts(258,20)` vs `src/fileWithError.ts:258:20`. The latter can be clicked through in VS Code's terminal to navigate to the line, the former cannot be. How can I get Turborepo to output the same thing as what I would expect when running the `yarn build` script in the package independently?

**Comment by mehulkar** [comment, marked as answer]
- It's a bit weird that the turbo output replaces `src/fileWithError.ts:258:20` with `src/fileWithError.ts(258,20)`. I'm not sure what's going on there, would you open an Issue for it, please?
- Removing the prefixes is being discussed and implemented in https://github.com/vercel/turbo/pull/3439
